lms/static/js/student_account/views/account_settings_factory.js

```diff
@@ -124,10 +124,11 @@
                             view: new AccountSettingsFieldViews.DropdownFieldView({
                                 model: userAccountModel,
                                 required: true,
-                                title: gettext('Country or Region'),
+                                title: gettext('Country of Full-Time Residence'),
```

Comment:
Per chat with @marcotuts, it's OK to drop the reference to "full-time", for clarity.